src/common/Parameter.cpp

```diff
@@ -236,6 +236,7 @@ bool Parameter::can_temposync()
     case ct_lforate:
     case ct_lforate_deactivatable:
     case ct_envtime:
+    case ct_envtime_deactivatable:
```

Comment:
Since this is specific for delay linking, perhaps rename this to ‘ct_envtime_linkable_delay’ since we special case the string in SGE (and would not want to use it for general deactivatable envtime)